README.md

```diff
@@ -1,12 +1,22 @@
-# bloom_analyses
+# Bloom Analysis
 
-A repository containing analyses used to perform bloom filtering
+Idenifies and removes sub OTUs which grow under room temperature storage conditions.
 
-# Install
+## Installion
+
+Download the repository from [GitHub](https://github.com/knightlab-analyses/bloom-analyses), and navigate into that directory. The repository can be installed by running the following commands in the linux terminal.
 
-To reproduce the analyses in the analyses in `ipynb`, run the following commands
 ```
-conda create -n bloom pip python=3 numpy scikit-bio jupyter seaborn matplotlib=1.5.1
+conda create -n bloom pip python=3 numpy jupyter seaborn matplotlib=1.5.1 statsmodels
+source activate bloom
+pip install scikit-bio==0.5.0
 pip install git+https://github.com/amnona/heatsequer.git
+pip install emperor --pre
 pip install -e .
-```
\ No newline at end of file
+```
+## Notebooks
```

Comment:
we explicitly mention in the submitted manuscript that there is a notebook which shows how to apply the filter.